charts/prometheus-blackbox-exporter/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: Prometheus Blackbox Exporter
 name: prometheus-blackbox-exporter
-version: 7.2.0
+version: 7.2.1
```

Comment:
new feature should raise minor version

Reply:
Do'h, yep that makes sense. I'll push up a change momentarily